Sources/ArrayExtensions.swift

```diff
@@ -19,6 +19,15 @@ public func ==<T: Equatable>(lhs: [T]?, rhs: [T]?) -> Bool {
 }
 
 extension Array {
+    
+    ///EZSE: Get a sub array from range of index
+    public func get(atIndex:Int,toIndex:Int) -> Array {
```

Comment:
think of `get(at range: ClosedRange<Int>) -> Array` 
And also you are missing checks for lower bound and upper bound
Thirdly you could implement it like `return self[range]`